mousestyles/classification/Classification.py

```diff
@@ -0,0 +1,72 @@
+from mousestyles import data
+import numpy as np
+from sklearn.grid_search import GridSearchCV
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.preprocessing import StandardScaler
+import pandas as pd 
+
+def RandomForest():
```

Comment:
Here I strongly suggest making the function RandomForest() to take a numpy array or pandas dataframe as the input, so that the user could specify whether to use a subset the data or use all data rows to fit the random forest. The function should also take the parameters like the number of trees, the number of features used in each split, and the tree size to be the input parameters so that it would be easier in future parameter tuning.